### PR TITLE
Changes to run PTS through hdfs

### PR DESCRIPTION
**Pull request by soumyaravi**
@chrismattmann Solves [#31](https://github.com/chrismattmann/pooled_time_series/issues/31)

### CODE REVIEW

File: src/main/bin/pooled-time-series-hadoop [hunk 1]
```diff
@@ -16,36 +16,57 @@
...
+hadoop fs -mkdir $HDFS_PTS_DIR
+hadoop fs -mkdir $TIME_SERIES_INPUT
+hadoop fs -mkdir $SIMILARITY_INPUT
+mkdir $PTS_TIME_SERIES_INPUT
+mkdir $PTS_SIMILARITY_INPUT
+
+echo "directory created"
+
+find $2 -name "*.mp4" > $PTS_TIME_SERIES_INPUT/original_videos.txt
+# hadoop fs -mv $TIME_SERIES_INPUT $PTS_TIME_SERIES_INPUT
```
Comment (chrismattmann) [inline src/main/bin/pooled-time-series-hadoop]: commented out - please remove.

File: src/main/bin/pooled-time-series-hadoop [hunk 2]
```diff
@@ -16,36 +16,57 @@
...
+hadoop fs -rm -r $HDFS_PTS_DIR/SimilarityCalc
+hadoop jar target/pooled-time-series-1.0-SNAPSHOT-jar-with-dependencies.jar gov.nasa.jpl.memex.pooledtimeseries.SimilarityCalculation $SIMILARITY_INPUT $HDFS_PTS_DIR/SimilarityCalc/ $MAIN_PTS_DIR/mean_dists.txt
 
-rm -rf $TIME_SERIES_INPUT
-rm -rf $SIMILARITY_INPUT
+hadoop fs -get $HDFS_PTS_DIR/SimilarityCalc/similarity_calc.txt $PTS_SIMILARITY_INPUT
+hadoop fs -rm -r $TIME_SERIES_INPUT
+hadoop fs -rm -r $SIMILARITY_INPUT
+#hadoop fs -rm -r $HDFS_PTS_DIR
+#rm -r $PTS_TIME_SERIES_INPUT
```
Comment (chrismattmann) [inline src/main/bin/pooled-time-series-hadoop]: remove

File: src/main/bin/pooled-time-series-hadoop [hunk 3]
```diff
@@ -16,36 +16,57 @@
...
+#rm -r $PTS_SIMILARITY_INPUT
```
Comment (chrismattmann) [inline src/main/bin/pooled-time-series-hadoop]: remove

**Comment by soumyaravi** [comment]
done.

**Comment by chrismattmann** [comment]
great job!

**Pull request merged**